.github/workflows/argilla-v1.yml

```
@@ -102,37 +102,56 @@ jobs:
   publish_release:
     name: Publish Release
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'release' }}
+    if: github.ref == 'refs/heads/releases/stable' || ${{ github.event_name }} == 'workflow_dispatch'
```

The final version will check the main branch.

```suggestion
    if: github.ref == 'refs/heads/main' || ${{ github.event_name }} == 'workflow_dispatch'
```